assets/js/app/Data.js

```diff
@@ -183,10 +183,12 @@ LocusZoom.Data.Source.prototype.getData = function(state, fields, outnames, tran
 };
 
 
-LocusZoom.Data.Source.prototype.parseResponse  = function(x, chain, fields, outnames, trans) {
-    var records = this.parseData(x.data || x, fields, outnames, trans);
-    var res = {header: chain.header || {}, body: records};
-    return res;
+LocusZoom.Data.Source.prototype.parseResponse = function(resp, chain, fields, outnames, trans) {
+    var json = resp;
+    if (typeof resp == "string"){ json = JSON.parse(resp); }
+    if (typeof json.data != "undefined"){ json = json.data; }
```

Comment:
Are there errors that this change is fixing? Are some of our responses not automatically being converted to JSON?